### DISCUSSION TITLE
Regarding the Education Benefits

### DISCUSSION THREAD

**Discussion by RamachandraKulkarni** [community/community#143785] [GitHub Education]
### Hi everyone,

I wanted to understand that it's been more than 5 days after approval of my application for the Education benefits. But I've not received any information regarding the educational benefits and I wanted to understand when would I be receiving the premium details?

**Comment by davevad93** [comment]
Hi @RamachandraKulkarni ,

The waiting time may vary depending on different factors, such as the volume of requests; it usually takes a few days, but sometimes it can take a little longer, even a couple of weeks.

When the benefits become available, the status label will turn purple and change to:

- `Congrats! Your request for @RamachandraKulkarni was approved and your benefits are now available.`

For further reference you can look at these discussions: #132790 , #131665 , #132470 , #133531, #134279, #135849, #136887, #138721, #139268

And also the [Student Developer Pack Application FAQs](https://github.com/orgs/community/discussions/111352) :

- [How long after I submit my application will it be approved?](https://github.com/orgs/community/discussions/111352#how-long-after-i-submit-my-application-will-it-be-approved)
- [How long after I’ve been approved will I receive my academic benefits?](https://github.com/orgs/community/discussions/111352#how-long-after-ive-been-approved-will-i-receive-my-academic-benefits)
